app/decorators/user_decorator.rb

```diff
@@ -8,4 +8,4 @@ def avatar_url
 
     "https://www.gravatar.com/avatar/#{hash}?s=60"
   end
-end
\ No newline at end of file
+end
```

Comment:
I'm under the impression VS Code wasn't doing its job when I was working on this 🤔

Reply:
Ok, I've fixed it now :)